crates/rolldown_binding/src/utils/normalize_binding_options.rs

```diff
@@ -117,45 +117,43 @@ fn normalize_globals_option(
 }
 
 fn normalize_es_target(target: Option<&Either<String, Vec<String>>>) -> ESTarget {
-  target
-    .map(|target| {
-      let targets = match target {
-        Either::A(target) => {
-          if target.contains(',') {
-            target.split(',').collect::<Vec<&str>>()
-          } else {
-            vec![target.as_str()]
-          }
-        }
-        Either::B(target) => target.iter().map(std::string::String::as_str).collect::<Vec<&str>>(),
-      };
-      for target in targets {
-        if target.len() <= 2 || !target[..2].eq_ignore_ascii_case("es") {
-          continue;
-        }
-        if target[2..].eq_ignore_ascii_case("next") {
-          return ESTarget::default();
-        }
-        if let Ok(n) = target[2..].parse::<usize>() {
-          return match n {
-            5 => ESTarget::ES5,
-            6 | 2015 => ESTarget::ES2015,
-            2016 => ESTarget::ES2016,
-            2017 => ESTarget::ES2017,
-            2018 => ESTarget::ES2018,
-            2019 => ESTarget::ES2019,
-            2020 => ESTarget::ES2020,
-            2021 => ESTarget::ES2021,
-            2022 => ESTarget::ES2022,
-            2023 => ESTarget::ES2023,
-            2024 => ESTarget::ES2024,
-            _ => continue,
-          };
+  target.map_or(ESTarget::ES2015, |target| {
```

Comment:
I was first ok with it and then thought that will it be more better to use the latest target by default?

The default behavior shouldn't downgrade code. It could only be done by configured by user manually.

Reply:
> If no target is provided at all, we default to ESTarget::ESNext.

This makes sense.